### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by drpaulralph**
Great job with the draft! Most of my edits are for maintaining consistency with the other standards and improving conciseness. Please see additional notes sent via email.

### CODE REVIEW

**Comment by Greg4cr** [comment]
FYI - changes I made today have introduced conflicts. We can just manually look over the proposed changes and choose how to implement them.

**Comment by drpaulralph** [comment]
I have literally no idea how to resolve merge conflicts in github, so you
should probably just edit as you see fit. Or, if you want, we can schedule
a call and you can talk me through it. I'd value the learning experience to
be honest.
*—*
*D. Paul Ralph, PhD*
Professor of Software Engineering
Faculty of Computer Science, Dalhousie University | 6050 University Avenue,
Room 216 | Halifax NS Canada | B3H 4R2
paul@paulralph.name | http://paulralph.name

Notice: This communication, including any attachments, is confidential and
may be protected by legal privilege. It is intended only for the person or
persons to whom it is addressed. If you have received this email in error,
please notify the sender by email or telephone.

**Comment by Greg4cr** [comment]
Eh, the "smart" thing for me to do would have been to handle the PR, THEN looked at the comments from the PDF, but oh well!

Not a big deal. We can untangle these. 

(FYI - what you'd do is click the "Resolve Conflicts" button on this page (https://github.com/Greg4cr/sbse-sigsoft-standard/pull/15) just above the comments block. It would give you a wed editor that would highlight the areas that conflict. For each of those, you can rewrite the segments to be a "new version". This would update your request with the changes you made. However, I'm happy to just take care of it sometime soon :) )

**Comment by drpaulralph** [comment]
Greg, thanks for explaining. It's probably better if you just take care of
it, but do reach out if you need any clarifications.
*—*
*D. Paul Ralph, PhD*
Professor of Software Engineering
Faculty of Computer Science, Dalhousie University | 6050 University Avenue,
Room 216 | Halifax NS Canada | B3H 4R2
paul@paulralph.name | http://paulralph.name

Notice: This communication, including any attachments, is confidential and
may be protected by legal privilege. It is intended only for the person or
persons to whom it is addressed. If you have received this email in error,
please notify the sender by email or telephone.

**Comment by Greg4cr** [comment]
FYI - I have resolved the conflicts manually. I recommend that (a) @drpaulralph do one more pass to make sure this looks OK (as I played with wording) and (b) everyone else do a quick pass to see if this look OK. I don't think there is anything contentious in here. Just make sure that the more concise wording doesn't lose any important meaning.

**Review by Greg4cr — APPROVED** [review]

**Comment by Greg4cr** [comment]
Resolved conflicts. Will merge, as nothing seems too contentious.
